### PR TITLE
Interface: Add smart scrolling to group member list

### PR DESCRIPTION
**Pull request by tylershuster**
This uses `react-window`, which is a minimal rewrite of `react-virtualized` mentioned by @galenwp  in #2983.
Basically it combines the "members" and "contacts" into an array, and given an index, either returns members first or contacts if it's shown all the members.

It's not pretty but...it works?

### CODE REVIEW

File: pkg/interface/src/apps/groups/components/lib/contact-sidebar.tsx [hunk 1]
```diff
@@ -166,8 +169,20 @@ export class ContactSidebar extends Component<ContactSidebarProps, ContactSideba
           >Channels</Link>
           {shareSheet}
           <h2 className="f9 pt4 pr4 pb2 pl4 gray2 c-default">Members</h2>
-          {contactItems}
-          {groupItems}
+          {this.memberbox.current ? <List
+            height={this.memberbox.current.getBoundingClientRect().height}
+            className="flex-auto"
+            itemCount={contactItems.length + groupItems.length}
+            itemSize={48}
```
Comment (tylershuster) [inline pkg/interface/src/apps/groups/components/lib/contact-sidebar.tsx:176]: `itemSize={48}` is based on a padding of 6 around a sigil of 32 which is not 48 damn it

File: pkg/interface/src/apps/groups/components/lib/contact-sidebar.tsx [hunk 2]
```diff
@@ -166,8 +169,20 @@ export class ContactSidebar extends Component<ContactSidebarProps, ContactSideba
           >Channels</Link>
           {shareSheet}
           <h2 className="f9 pt4 pr4 pb2 pl4 gray2 c-default">Members</h2>
-          {contactItems}
-          {groupItems}
+          {this.memberbox.current ? <List
```
Comment (tylershuster) [inline pkg/interface/src/apps/groups/components/lib/contact-sidebar.tsx:172]: We see if the parent has been mounted yet first so we can set the height to that

**Comment by matildepark** [comment]
On ~bitbet-bolbel I get ... nothing?

image

**Comment by matildepark** [comment]
Also, man, this should probably also be in `GroupView` because the Group Settings stalls for about ten seconds on UC.

**Comment by tylershuster** [comment]
> On ~bitbet-bolbel I get ... nothing?

*On* ~bitbet-bolbel? Like...as? This is UC, right? What steps did you take to get this failure?

**Comment by matildepark** [comment]
> _On_ ~bitbet-bolbel? Like...as?

Yes.

> What steps did you take to get this failure?

1. Sign in.
2. Go to Groups.
3. Click Urbit Community. Blank.
4. Click "Add to Group" or refresh the page — members show up.

**Comment by tylershuster** [comment]
Weird, but confirmed.

**Comment by tylershuster** [comment]
Ok check it. For the `GroupView`, this may be something to run past @urcades because I don't know that the current approach (setting it to 500px):

pic

is exactly A  E  S  T  H  E  T  I  C  .

Basically it's a "window" into a list. It's not exactly...if something comes into screen, load it. Which I don't know is possible. LMK if you know of a library, but this seems to be how they're all made.

**Comment by matildepark** [comment]
Wanna squash these commits?

**Pull request merged**